setup.py

```diff
@@ -57,90 +31,16 @@
     exec(f.read(), ABOUT)
 
 
-#
-# Utilities
-#
-
-class VerifyVersionCommand(install):
-    """Custom command to verify that the git tag matches our version"""
-    description = 'verify that the git tag matches our version'
-
-    def run(self):
-        tag = os.getenv('CIRCLE_TAG')
-        if tag.startswith('v'):
-            tag = tag[1:]
-
-        version = ABOUT['__version__']
-        if version.startswith('v'):
-            version = version[1:]
-
-        if tag != version:
-            info = "Git tag: {0} does not match the version of this app: {1}".format(
-                os.getenv('CIRCLE_TAG'), ABOUT['__version__']
-            )
-            sys.exit(info)
-
-
-class PostDevelopCommand(develop):
-    """
-    Post-installation for development mode.
-    Execute manually with python setup.py develop or automatically included with
-    `pip install -e . -r dev-requirements.txt`.
-    """
-
-    def run(self):
-        """development setup scripts (pre-requirements)"""
-        develop.run(self)
-
-
-#
-#  Requirements
-#
-
 def read_requirements(path):
     with open(BASE_DIR / path) as f:
-        _pipenv_flags, *lines = f.read().split('\n')
-
-    # TODO remove when will be no more git dependencies in requirements.txt
-    # Transforms VCS requirements to PEP 508
-    requirements = []
-    for line in lines:
-        if line.startswith('-e git:') or line.startswith('-e git+') or \
-                line.startswith('git:') or line.startswith('git+'):
-            # parse out egg=... fragment from VCS URL
-            parsed = urlparse(line)
-            egg_name = parsed.fragment.partition("egg=")[-1]
-            without_fragment = parsed._replace(fragment="").geturl()
-            requirements.append(f"{egg_name} @ {without_fragment}")
-        else:
-            requirements.append(line)
-
-    return requirements
+        return f.read().split("\n")
 
 
 INSTALL_REQUIRES = read_requirements("requirements.txt")
 DEV_REQUIRES = read_requirements("dev-requirements.txt")
 
-BENCHMARK_REQUIRES = [
-    'pytest-benchmark'
-]
-
-DEPLOY_REQUIRES = [
-    'bumpversion',
-    'ansible',
-    'twine',
-    'wheel'
-]
-
-URSULA_REQUIRES = ["sentry-sdk"]
-
 EXTRAS = {
-
-    # Admin
-    "dev": DEV_REQUIRES + URSULA_REQUIRES,
-    "benchmark": DEV_REQUIRES + BENCHMARK_REQUIRES,
-    "deploy": DEPLOY_REQUIRES,
-    "ursula": URSULA_REQUIRES,
```

Comment:
This "ursula" extra is used by the Dockerfile for building `nucypher/nucypher`...if we are getting rid of it, we should update the Dockerfile.

Reply:
✔️